yarn.lock

```diff
@@ -0,0 +1,10776 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
singlee-spa-angular currently uses npm, not yarn. There is a package-lock.json file in source control and in this PR that was changed. Are you proposing changing to yarn? Or why did you modify both the package-lock and the yarn.lock?

Reply:
> Are you proposing changing to yarn?

Yes. I do prefer yarn over npm same as you. I'm also awaiting the yarn 2. That will significantly speed up the installation process in the future when it comes out of beta. What are your thoughts? I forgot to remove `package-lock`.

Reply:
👍 let's remove the package-lock.json and switch to yarn 1. Yarn 2 is quite a leap, with removal of the node_modules directory, so I'd prefer holding off on that one.